pareto/utilities/get_data.py

```diff
@@ -53,25 +163,34 @@ def _read_data(_fname, _set_list, _parameter_list):
 
     _df_parameters = pd.read_excel(
```

Comment:
Since we are reading in all tabs with the call to `pd.read_excel()`, it seems unnecessary to call this function twice. I recommend reformatting the code to call `pd.read_excel()` only once, and then extract the sets and parameters.

Reply:
Looking at this again, the way it was designed is intentional. Sets and parameters are read in with different settings of `pd.read_excel` and are subsequently processed a bit differently.